spec/system/signed_in_user_creates_a_recipe_spec.rb

```diff
@@ -1,21 +1,39 @@
 # frozen_string_literal: true
 
-require "rails_helper"
-require "support/features/clearance_helpers"
+require 'rails_helper'
+require 'support/features/clearance_helpers'
 
-RSpec.feature "A signed in user creates a recipe" do
-  scenario "with valid parameters" do
+RSpec.feature 'A signed in user creates a recipe' do
+  scenario 'with valid parameters, but no picture' do
     user = create(:user)
     recipe = FactoryBot.build(:recipe)
 
     sign_in_with user.email, user.password
-    visit "/"
-    click_link I18n.t("layouts.application.new_recipe")
-    fill_in "Title", with: recipe.title
-    fill_in "Ingredients", with: recipe.ingredients
-    fill_in "Instructions", with: recipe.instructions
-    click_button "Create"
+    visit '/'
+    click_link I18n.t('layouts.application.new_recipe')
+    fill_in 'Title', with: recipe.title
+    fill_in 'Ingredients', with: recipe.ingredients
+    fill_in 'Instructions', with: recipe.instructions
+    click_button 'Create'
 
-    expect(page).to have_current_path("/")
+    expect(page).to have_current_path('/')
+  end
+
+  scenario 'and includes a picture' do
+    user = create(:user)
+    recipe = FactoryBot.build(:recipe)
+
+    sign_in_with user.email, user.password
+    visit '/'
+    click_link I18n.t('layouts.application.new_recipe')
+    fill_in 'Title', with: recipe.title
+    fill_in 'Ingredients', with: recipe.ingredients
+    fill_in 'Instruction', with: recipe.instructions
+    recipe.
```

Comment:
Should this be [`#attach_file`](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FActions:attach_file)? Otherwise this is exercising ActiveStorage itself but not your app.

Reply:
Yep, I agree! I initially tried `attach_file`, but was having trouble getting it to work when writing this. I will look into it more.

Reply:
I mentioned this in a comment above, but I managed to utilize `attach_file` here as well. 🎉 Thanks for pointing that out.